### DISCUSSION TITLE
any plan to enable cpptools with personal rules?

### DISCUSSION THREAD

**Discussion by 64Teenage** [microsoft/vscode-cpptools#9676] [Q&A]
any plan to enable cpptools with personal rules?

**Comment by Colengms** [comment]
Hi @64Teenage .  Could you clarify what you mean by "personal rules"?

**Reply by 64Teenage** [reply]
I want to get a red squiggle under the code such as 
```Cpp
memset(pBuffer, 1, sizeof(char)*MAX_BYTES);
```
is there any iterface to implement the rule

**Reply by sean-mcmanus** [reply]
You can get a yellow squiggle for this case via running Code Analysis: 
![image](https://user-images.githubusercontent.com/19859882/183537239-60e46ec3-d970-4d88-bbd6-bf965ea0438e.png)

Otherwise, it might be tracked for IntelliSense via https://github.com/microsoft/vscode-cpptools/issues/828.
